.github/ISSUE_TEMPLATE/docs.yml

```diff
@@ -0,0 +1,26 @@
+name: 📄 Documentation issue
+description: Found an issue in the documentation? You can use this issue template
+title: '[DOCS] <description>'
+labels: ['documentation']
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the question or issue, also include what you tried and what didn't work
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Please add screenshots if applicable
+    validations:
+      required: false
+  - type: textarea
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this issue?
+    validations:
+      required: false
```

Comment:
```suggestion
      required: false

```

Reply:
🤔 Hey could I know what's the issue over here ?

Reply:
End line